components/eventing-controller/api/v1alpha1/subscription_types.go

```diff
@@ -4,14 +4,20 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// Webhook defines the Webhook called by an active subscription in BEB
+// WebhookAuth defines the Webhook called by an active subscription in BEB
 type WebhookAuth struct {
-	Type         string   `json:"type,omitempty"`
-	GrantType    string   `json:"grantType"`
-	ClientId     string   `json:"clientId"`
-	ClientSecret string   `json:"clientSecret"`
-	TokenUrl     string   `json:"tokenUrl"`
-	Scope        []string `json:"scope,omitempty"`
+	// +optional
```

Comment:
could we also add descriptions to those properties?

Reply:
The comment [here](https://github.com/kyma-project/kyma/pull/11447/files/12bb85ec4de2c7228b6022340755a1f85ecaa6ae#diff-15ef6a41dff50cd9aa8cd717dba2729f245f21b5ceeae3527b268ec7804256d2R7) is explanatory hence left it intentionally to make it less verbose. I added them anyway.